src/sensors/MAX31865.h

```diff
@@ -0,0 +1,84 @@
+#pragma once
+
+// Adapted from https://github.com/adafruit/Adafruit_MAX31865
+
+/***************************************************
+  This is a library for the Adafruit PT100/P1000 RTD Sensor w/MAX31865
+  Designed specifically to work with the Adafruit RTD Sensor
+  ----> https://www.adafruit.com/products/3328
+  This sensor uses SPI to communicate, 4 pins are required to
+  interface
+  Adafruit invests time and resources providing this open source code,
+  please support Adafruit and open-source hardware by purchasing
+  products from Adafruit!
+  Written by Limor Fried/Ladyada for Adafruit Industries.
+  BSD license, all text above must be included in any redistribution
+ ****************************************************/
+
+#include <IO/SPI.h>
+#include <cstdint>
+#define MAX31865_CONFIG_REG 0x00
```

Comment:
I think using a `constexpr` or `enum class` for configuration flags would be benefitial for clarity compared to macros.